Library/Formula/simple-tiles.rb

```diff
@@ -1,19 +1,37 @@
-require 'formula'
-
 class SimpleTiles < Formula
-  homepage 'http://propublica.github.io/simple-tiles/'
-  url 'https://github.com/propublica/simple-tiles/archive/v0.5.0.tar.gz'
-  sha1 'dde1359132f29e56e01596cbab58b1ed85d2de08'
+  homepage "https://propublica.github.io/simple-tiles/"
+  url "https://github.com/propublica/simple-tiles/archive/v0.6.0.tar.gz"
+  sha256 "336fdc04c34b85270c377d880a0d4cc2ac1a9c9e5e4c3d53e0322d43c9495ac9"
 
-  head 'https://github.com/propublica/simple-tiles.git'
+  head "https://github.com/propublica/simple-tiles.git"
 
   depends_on "pkg-config" => :build
   depends_on "cairo"
   depends_on "gdal"
   depends_on "pango"
 
+  resource "countries" do
+    url "http://www.naturalearthdata.com/http//www.naturalearthdata.com/download/10m/cultural/ne_10m_admin_0_countries.zip"
+    sha256 "1cfe3d8f9fb0c40fd4424ef6e7b56829984df103fd7fa4767f34ebd5399e0277"
+  end
+
+  resource "loss" do
+    url "http://propublica.s3.amazonaws.com/projects/simple-tiles/test_data/loss_1932_2010.zip"
+    sha256 "5b9651ef4cec69b78e2c86f9a107bf216e2192746f8e83e7f1cac61f0bf0c60d"
+  end
+
   def install
     system "./configure", "--prefix=#{prefix}"
-    system "make install"
+    system "make", "install"
+    prefix.install "build/test"
+  end
+
+  test do
+    cp prefix/"test/api", testpath
+    resources.each do |r|
+      (testpath/"data").install r
+    end
+    system testpath/"api"
+    File.exist? "out.png"
```

Comment:
Wouldn't it still need one of the sample data packages for this line `simplet_vector_layer_t *layer = simplet_map_add_vector_layer(map, "PG:dbname=simple_tiles_roads");`?